package/opt/hassbian/suites/appdaemon.sh

```diff
@@ -53,34 +60,37 @@ sync
 
 echo "Starting AppDaemon service"
 systemctl start appdaemon@homeassistant.service
-if [ "$ACCEPT" != "true" ]; then
-  if [ -f "/usr/sbin/samba" ]; then
-  	read -p "Do you want to add samba share for AppDaemon configuration? [N/y] : " SAMBA
-  	if [ "$SAMBA" == "y" ] || [ "$SAMBA" == "Y" ]; then
-  		echo "Adding configuration to samba..."
-  		echo "[appdaemon]" | tee -a /etc/samba/smb.conf
-  		echo "path = /home/homeassistant/appdaemon" | tee -a /etc/samba/smb.conf
-  		echo "writeable = yes" | tee -a /etc/samba/smb.conf
-  		echo "guest ok = yes" | tee -a /etc/samba/smb.conf
-  		echo "create mask = 0644" | tee -a /etc/samba/smb.conf
-  		echo "directory mask = 0755" | tee -a /etc/samba/smb.conf
-  		echo "force user = homeassistant" | tee -a /etc/samba/smb.conf
-  		echo "" | tee -a /etc/samba/smb.conf
-  		echo "Restarting Samba service"
-  		sudo systemctl restart smbd.service
-  	fi
-  fi
+
+if [ "$SAMBA" == "y" ] || [ "$SAMBA" == "Y" ]; then
+	echo "Adding configuration to samba..."
+	echo "[appdaemon]" | tee -a /etc/samba/smb.conf
+	echo "path = /home/homeassistant/appdaemon" | tee -a /etc/samba/smb.conf
+	echo "writeable = yes" | tee -a /etc/samba/smb.conf
+	echo "guest ok = yes" | tee -a /etc/samba/smb.conf
+	echo "create mask = 0644" | tee -a /etc/samba/smb.conf
+	echo "directory mask = 0755" | tee -a /etc/samba/smb.conf
+	echo "force user = homeassistant" | tee -a /etc/samba/smb.conf
+	echo "" | tee -a /etc/samba/smb.conf
+	echo "Restarting Samba service"
+	sudo systemctl restart smbd.service
 fi
 
-echo
-echo "Installation done."
-echo
-echo "You may find the appdaemon configuration files in:"
-echo "/home/homeassistant/appdaemon"
-echo "To continue have a look at http://appdaemon.readthedocs.io/en/latest/"
-echo
-echo "If you have issues with this script, please say something in the #devs_hassbian channel on Discord."
-echo
+validation=$(ps -ef | grep -v grep | grep appdaemon | wc -l)
+if [ "$validation" != "0" ]; then
+	echo
+	echo -e "\e[32mInstallation done..\e[0m"
+	echo
+  echo "You will find the appdaemon configuration files in:"
```

Comment:
Formatting white space missing here and two line more.

Reply:
Not sure how :/

Reply:
It's a "TAB" that's missing before the `echo`

Reply:
Yeah, I see that is missing, not sure how that happened :)